src/modules/keyboardmanager/ui/BufferValidationHelpers.cpp

```diff
@@ -117,8 +117,12 @@ namespace BufferValidationHelpers
                     }
                     else
                     {
-                        // warn and reset the drop down
-                        errorType = KeyboardManagerHelper::ErrorType::ShortcutOneActionKey;
+                        //None is allowed when it is the only choice (Disabling keys)
+                        if (dropDownIndex != 0)
```

Comment:
Just to make this cleaner, it would be better to check `if (dropDownCount == 1)`. This is equivalent since we are checking that this is the last drop down at line 85, but just to make it clearer to anyone reading the code. Also just to make it consistent with the previous conditions this condition of selecting Disable key should be available only if `isHybridControl` is true. So I think the best way to change it is `if (dropDownCount != 1 || !isHybridControl)`

Reply:
Done in 2b3c69f.